rickshaw-run

```diff
@@ -96,6 +96,7 @@ my $workshop_roadblock_opt = "";
 my %utility_configs;
 my %tools_configs;
 my @tools_params;
+my $default_tool_userenv = 'fedora39';
```

Comment:
I wonder if this should be in the settings JSON file.

Reply:
moving it